### DISCUSSION TITLE
Redirecting to specific chart when filters are applied

### DISCUSSION THREAD

**Discussion by CeelamkotiVinod** [apache/superset#23620] [General]
I have been using Superset for two years and have a requirement to redirect the user to a specific chart(other tab) within a dashboard when they click on a specific value in a table(a particular chart). It is more like a **clickable links**.

I have used **cross filters** but it is not redirecting anywhere when applied. But filters are applying within the dashboard wherever that particular dataset is used.

Is is possible to implement this feature?

If so, any insights will be appreciable!

Thanks in advance.

**Comment by apole** [comment]
I am trying the same thing , by adding column  like this.   Idea is when clicked on this it should navigate to specified tab or dashboard. <a target="_self" href=http://<superset server>/superset/dashboard/p/mwasnepbA89G/?standalone=3&native_filters=>

This does not work when dashboard is embedded.  It takes me back to the Login page.
